js/containers/DeployModal.jsx

```diff
@@ -112,6 +111,16 @@ const mapStateToProps = function(state) {
 		return state.approval.approved || state.approval.bypassed;
 	}
 
+	let active_step = 0;
+	if (window.location.hash) {
+		active_step = parseInt(window.location.hash.substring(1), 10);
+	}
+
+	let is_open = false;
+	if (ownProps.params) {
+		is_open = ownProps.params.id !== null || typeof (ownProps.params.id) !== 'undefined';
```

Comment:
you are doing a three combination comparison if here, just hidden as one `if` and a double `true comparison` assignment. Change to either use only an if or the comparison to make it more readable.
```
if (typeof (ownProps.params.id) !== 'undefined' && ownProps.params.id !== null) {
   is_open = true;
}
```
or 

```
is_open = typeof (ownProps.params.id) !== 'undefined' && ownProps.params.id !== null;
```